### PR TITLE
Aura example

### PR DESCRIPTION
**Pull request by mneedham**

### CODE REVIEW

**Comment by mneedham** [comment]
This is how it'll look on the Aura console:

image

**Comment by mneedham** [comment]
Updated screenshot:

image

File: examples/src/main/java/org/neo4j/docs/driver/DriverIntroductionExample.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    public static void main(String... args) throws Exception {
+        // Aura uses the "neo4j" protocol
+        String boltUrl = "%%BOLT_URL_PLACEHOLDER%%";
+
+        String user = "<Username for Neo4j Aura database>";
+        String password = "<Password for Neo4j Aura database>";
+
+        // Aura queries use an encrypted connection
+        Config config = Config.builder().withEncryption().build();
```
Comment (gjmwoods) [inline examples/src/main/java/org/neo4j/docs/driver/DriverIntroductionExample.java:107]: just a thought about dropping this config in favor of `neo4j+s` scheme

**Review by gjmwoods — APPROVED** [review]
looks like a nice clear example to me. cheers!

File: examples/src/main/java/org/neo4j/docs/driver/DriverIntroductionExample.java (hunk 1 above)
Comment (mneedham) [reply examples/src/main/java/org/neo4j/docs/driver/DriverIntroductionExample.java:107]: I didn't know about that scheme! But yes we can use that. I've updated the example

**Pull request merged**